### PR TITLE
[Feat]#39-DetailViewModel수정

### PR DESCRIPTION
**Pull request by qkwmapel**
## What is the PR? 🔍

 - ViewModel수정

## Related Issues 💭

- Resolved: #39

### CODE REVIEW

File: Dokgi/View/ ParagraphDetailView/ParagraphDetailViewController.swift [hunk 1]
```diff
@@ -156,31 +149,22 @@ class ParagraphDetailViewController: UIViewController {
         containerView.keywordTextField.rx.controlEvent(.editingDidEnd).subscribe(with: self) { (self, _) in
             if let text = self.containerView.keywordTextField.text {
                 self.viewModel.addDetailKeyword(keyword: text)
-                self.containerView.keywordCollectionView.reloadData()
+                self.containerView.keywordTextField.text = ""
+            }
+        }.disposed(by: disposeBag)
+        
+        viewModel.detailParagraph.map{ $0.keywords }.bind(to: containerView.keywordCollectionView.rx.items(cellIdentifier: KeywordCollectionViewCell.identifier,
+                   cellType: KeywordCollectionViewCell.self )) {row, data, cell in
```
Comment (suri0000) [inline Dokgi/View/ ParagraphDetailView/ParagraphDetailViewController.swift:157]: 띄어쓰기

**Review by suri0000 — APPROVED** [review]
고생하셨습니다!!

**Review by Imhnjng — APPROVED** [review]
수고하셨습니다!

**Review by myhan601 — APPROVED** [review]
수고하셨습니다!

**Pull request merged**